modules/cudf/test/series/string-tests.ts

```diff
@@ -36,19 +35,19 @@ const data = [
 describe.each([['foo'], [/foo/], [/foo/ig]])('Series regex search (pattern=%p)', (pattern) => {
   test('containsRe', () => {
     const expected = [true, true, true, true, true, false, false, true, true];
-    const s        = StringSeries.new(Utf8Vector.from(data));
+    const s        = Series.new(data as any);
```

Comment:
`new` added `string[]` to the type for `data`, why is the `as any` needed?

Reply:
fixed this, as any is not needed now